test/test_ops.py

```diff
@@ -54,6 +57,12 @@ def test_supported_dtypes(self, device, dtype, op):
         sample = samples[0]
         op(*sample.input, *sample.args, **sample.kwargs)
 
+    # Verifies that ops do not have an entry in
+    # `method_tests` (legacy testing infra).
+    @onlyCPU
+    @ops(op_db, allowed_dtypes=[torch.float32])
```

Comment:
Smart implementation to support skips so this PR doesn't have to update all the sample inputs, too. If there were no skips then this could probably just compare the op_db and method_tests() directly.